src/tasks/tasks.proj

```diff
@@ -49,7 +49,7 @@
     <!-- Include both the project file and its sources as an input. -->
     <ItemGroup>
       <TasksSrc Include="%(ProjectReferenceWithConfiguration.Identity)" />
-      <TasksSrc Include="%(ProjectReferenceWithConfiguration.RelativeDir)%(ProjectReferenceWithConfiguration.RecursiveDir)**\*.cs" />
+      <TasksSrc Include="%(ProjectReferenceWithConfiguration.RelativeDir)%(ProjectReferenceWithConfiguration.RecursiveDir)**\*" />
```

Comment:
That probably includes the .csproj file twice because of the line above.

Reply:
yeah, does it matter though? I guess we could simplify the whole thing to `$(MSBuildThisFileDirectory)**\*` so we don't even need to rely on the `ProjectReferenceWithConfiguration `

Reply:
That would include everything which we don't want as some P2Ps are condition per TargetOS. I would just remove the line above.

Reply:
Ok.